ethsigner/signer/multiplatform/src/test/resources/metadata-toml-configs/p.password

```diff
@@ -0,0 +1 @@
+pantheon
```

Comment:
nit: Do we need a whole file for this, or can we have the test harness make a temp file?

Reply:
Why `pantheon`?